test/extended/authorization/podsecurity_admission.go

```diff
@@ -189,14 +193,302 @@ var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
+					ns.ObjectMeta.Labels["security.openshift.io/scc.podSecurityLabelSync"] = testCase.podSecurityLabelSync
+				}
+
+				// create the test namespace without any PSa labels; podSecurityLabelSync label according to each test case
+				ns, err := createOrUpdateNamespace(ctx, oc, ns, metav1.CreateOptions{})
+				o.Expect(err).NotTo(o.HaveOccurred())
+
+				// assert expectations after creation depending on the podSecurityLabelSync label
+				o.Expect(ns.GetLabels()[psapi.EnforceLevelLabel]).To(o.Equal(testCase.expectedPSaLevelAfterCreate))
+				o.Expect(ns.GetLabels()[psapi.EnforceVersionLabel]).To(o.Equal(testCase.expectedPSaVersionAfterCreate))
+				o.Expect(ns.GetLabels()[psapi.AuditLevelLabel]).To(o.Equal(testCase.expectedPSaLevelAfterCreate))
+				o.Expect(ns.GetLabels()[psapi.AuditVersionLabel]).To(o.Equal(testCase.expectedPSaVersionAfterCreate))
+				o.Expect(ns.GetLabels()[psapi.WarnLevelLabel]).To(o.Equal(testCase.expectedPSaLevelAfterCreate))
+				o.Expect(ns.GetLabels()[psapi.WarnVersionLabel]).To(o.Equal(testCase.expectedPSaVersionAfterCreate))
+
+				// test overwriting the PSa labels
+				ns.ObjectMeta.Labels[psapi.EnforceLevelLabel] = testCase.expectedPSaLevelAfterUpdate
+				ns.ObjectMeta.Labels[psapi.EnforceVersionLabel] = testCase.expectedPSaVersionAfterUpdate
+				ns.ObjectMeta.Labels[psapi.AuditLevelLabel] = testCase.expectedPSaLevelAfterUpdate
+				ns.ObjectMeta.Labels[psapi.AuditVersionLabel] = testCase.expectedPSaVersionAfterUpdate
+				ns.ObjectMeta.Labels[psapi.WarnLevelLabel] = testCase.expectedPSaLevelAfterUpdate
+				ns.ObjectMeta.Labels[psapi.WarnVersionLabel] = testCase.expectedPSaVersionAfterUpdate
+				ns, err = createOrUpdateNamespace(ctx, oc, ns, metav1.UpdateOptions{})
+				o.Expect(err).NotTo(o.HaveOccurred())
+
+				// assert expectations after the update based on the podSecurityLabelSync label
+				o.Expect(ns.GetLabels()[psapi.EnforceLevelLabel]).To(o.Equal(testCase.expectedPSaLevelAfterUpdate))
+				o.Expect(ns.GetLabels()[psapi.EnforceVersionLabel]).To(o.Equal(testCase.expectedPSaVersionAfterUpdate))
+				o.Expect(ns.GetLabels()[psapi.AuditLevelLabel]).To(o.Equal(testCase.expectedPSaLevelAfterUpdate))
+				o.Expect(ns.GetLabels()[psapi.AuditVersionLabel]).To(o.Equal(testCase.expectedPSaVersionAfterUpdate))
+				o.Expect(ns.GetLabels()[psapi.WarnLevelLabel]).To(o.Equal(testCase.expectedPSaLevelAfterUpdate))
+				o.Expect(ns.GetLabels()[psapi.WarnVersionLabel]).To(o.Equal(testCase.expectedPSaVersionAfterUpdate))
+
+				// add a role to the default SA
+				role, err := bindRoleToDefaultSA(ctx, oc, ns.Name)
+				o.Expect(err).NotTo(o.HaveOccurred())
+
+				// set some SCCs on the role and test PSa label outcome
+				for scc, psaLabel := range sccToPSa {
+					err = setSCCPermissions(ctx, oc, role, scc)
+					o.Expect(err).NotTo(o.HaveOccurred())
+
+					ns, err = waitAndGetNamespace(ctx, oc, ns.Name)
+					o.Expect(err).NotTo(o.HaveOccurred())
+
+					if testCase.sccChangesPSa {
```

Comment:
```go
expectedLevelLabel := testCase.expectedPSaLevelAfterUpdate
if testCase.sccChangesPSa {
    expectedLevelLabel = psaLabel
}
<checks here>
```

Reply:
fixed in 3de47a4c0f95c5ecdccabd34cb604bfda986bdae